Resources/config/services.xml

```diff
@@ -233,6 +233,11 @@
             <tag name="jms_serializer.deserialization_visitor" format="xml" />
         </service>
 
+        <service id="jms_serializer.cache.cache_clearer" class="JMS\SerializerBundle\Cache\CacheClearer" public="false">
+            <argument type="service" id="jms_serializer.metadata.cache"/>
```

Comment:
I think this is missing `on-invalid="null` entry, as disabling cache (with `cache: none`) is broken now.

The constructor should be updated to accept null then. :)

Reply:
I'll fix it ASAP

Reply:
I have already opened a PR #850., would be nice if you could review it. 👍